.github/workflows/reusable_build.yml

```diff
@@ -41,15 +41,8 @@ jobs:
       - name: 'Build tarantool packages for ${{ env.OS }}(${{ env.DIST }})'
         id: run_build
         env:
-          # Our testing expects that the init process (PID 1) will
```

Comment:
Looks like it was just a copy-past from the `pack_and_deploy` action. Now it is not needed  due to a few reasons:
- testing is off
- `DEBIAN_FRONTEND` is now not needed because we switched to RWS

Reply:
Seems correct for me too.